timesheet_automatic_remind/models/timesheet_employee_remind.py

```diff
@@ -0,0 +1,291 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import datetime
+import time
+import pytz
+from openerp import fields, models, api
+
+
+class TimesheetReport(models.TransientModel):
+    _name = "timesheet.employee.remind.date"
+    _description = "Timesheet Employee Remind Date"
+
+    reminder_date = fields.Char(
+        string='Reminder Date',
+    )
+
+    employee_remind_id = fields.Many2one(
+        string='Reminding Employee',
+        comodel_name='timesheet.employee.remind'
+    )
+
+
+class TimesheetReport(models.TransientModel):
+    _name = "timesheet.employee.remind"
+    _description = "Timesheet Employee Remind Service"
+    _inherit = ['mail.thread']
+
+    MONDAY = 0
```

Comment:
@Zhouhao12345 since those numbers are documented standard python "constants", I don't think it's necessary to re-define them. Especially since you don't use them, except `WEEKEND`. Please remove the other ones.